controller/common/common_types.py

```diff
@@ -0,0 +1,7 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class ObjectIds:
+    internal_id: str = ''
+    uid: str = ''

```

Comment:
could this be in the same file as `class Volume`? we can always move them together wherever we see fit

Reply:
no, its creating some loop problems